bazel/foreign_cc/curl.patch

```diff
@@ -67,14 +67,3 @@ index 6a1a6fe8e..777ee122f 100644
 +  endforeach()
  endif()
  
-@@ -619,5 +627,9 @@ endif()
- 
- # Check for idn
--check_library_exists_concat("idn2" idn2_lookup_ul HAVE_LIBIDN2)
-+option(USE_LIBIDN2 "Use libidn2 for IDN support" ON)
-+set(HAVE_LIBIDN2 OFF)
-+if(USE_LIBIDN2)
-+  check_library_exists_concat("idn2" idn2_lookup_ul HAVE_LIBIDN2)
-+endif()
- 
- # Check for symbol dlopen (same as HAVE_LIBDL)
```

Comment:
This looks correct, now incorporated in the release with https://github.com/curl/curl/commit/83f1ca69290f9d6198fe4013488408f94736c5dc

The corresponding USE_WIN32_IDN change defaults to off, which is our desired omission.